utils/vast-maker.js

```diff
@@ -191,35 +216,39 @@ function AttachPodAds(vast4, podAds, params) {
           idRegistry: 'test-ad-id.eyevinn',
           idValue: encodeURIComponent(`${podAds[i].universalId}${i + 1}`),
         }
```

Comment:
Whole PR is great. Nice attention to detail with the adId attribute. Weirdly enough IAB's own VAST validator claims that adID is not allowed but it clearly states in the VAST 3 specs that you canhave it. 

Anyway, ONE small change. The duplicated universalAdId must have been a typo from long ago. Do not remeber setting it on purpose, but there is no need for 2 universal ad ids in the same creative. 
I see that you perserved the extra `addUniversalAdId()` but you can go ahead and remove that

Reply:
Which one of these is the correct one then?

https://github.com/Eyevinn/test-adserver/blob/b924921c658caf814a66af7b6c962f36da39b8a1/utils/vast-maker.js#L154-L164

The thing that differs is the `idRegistry`.